packages/vm/src/errors.rs

```diff
@@ -99,28 +102,120 @@ impl From<wasmer_runtime_core::cache::Error> for VmError {
     }
 }
 
+impl From<wasmer_runtime_core::error::CompileError> for VmError {
+    fn from(original: wasmer_runtime_core::error::CompileError) -> Self {
+        make_compile_err(format!("Wasmer compile error: {:?}", original))
```

Comment:
But these lines, isn't that a bit much indirection? I think

`CompileErr { msg: format!("Wasmer compile error: {:?}", original) }.build()`

should be clearer, and not much longer.

Reply:
Jap, less indirection. Fine for me, will change

Reply:
Ah, I guess my idea was: `fn from(original: wasmer_runtime_core::error::CompileError) -> Self` is not tested, but `make_compile_err` is. So the currect code keeps the untested area as small as possible.